news/_posts/public-oss-demo/2021-04-29-public-oss-demo-4-2021.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title:  "PrestaShop Public OSS Demo - April 2021 session"
+subtitle: "We held the fourth session of the open source project's public demo"
+date:   2021-04-29 12:00:00
+authors: [ MathieuFerment ]
+icon: icon-video-camera
+tags: [contribute, event, community]
+---
+
+The project's fourth monthly public demo session was held and [streamed live](https://youtu.be/VKb39T_6WHU) on April 28, 2021.
+
+The session's agenda was as follows:
+
+- [00:00](https://youtu.be/VKb39T_6WHU) Waiting for the stream to begin
+- [00:20](https://youtu.be/VKb39T_6WHU?t=20) Beginning of the session, introduction
+- [00:51](https://youtu.be/VKb39T_6WHU?t=51) Presentation of the agenda
+- [01:31](https://youtu.be/VKb39T_6WHU?t=91) Event rules
+- [02:25](https://youtu.be/VKb39T_6WHU?t=145) Project announcements
+- [03:30](https://youtu.be/VKb39T_6WHU?t=210) Demonstration #1 - Multistore news in 1.7.8 by [@marionf](https://github.com/marionf)
+- [15:45](https://youtu.be/VKb39T_6WHU?t=945) Demonstration #2 - Blockwishlist v2.0.0 by [@lolath-presta](https://github.com/lolath-presta) and [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+- [19:17](https://youtu.be/VKb39T_6WHU?t=1157) Small network outage
```

Comment:
I dont know if it's necessary to mention it. However the timestamp allows people to skip the outage and go back to the demo

Reply:
I agree, but couldn't we use Youtube for this? adding timestamp themes, so we don't need to add it into a post?

Reply:
This is an interesting idea.

However this would make the link between the Public OSS Demos and YouTube stronger. Today we only use YouTube as a streaming channel. I am wondering if we should offer an alternative format for people and if yes, timestamps should not be controlled from within YouTube.

Reply:
Issue created https://github.com/PrestaShop/prestashop.github.io/issues/889